base_phone_rate_import_bandwith/README.rst

```diff
@@ -0,0 +1,90 @@
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fvertical--isp-lightgray.png?logo=github
+    :target: https://github.com/OCA/vertical-isp/tree/12.0/base_phone_rate_import_bandwith
+    :alt: OCA/vertical-isp
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/vertical-isp-12-0/vertical-isp-12-0-base_phone_rate_import_bandwith
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/158/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows you to load and update the phone rates using the file provided by Bandwidth (https://www.bandwidth.com).
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+This module needs no additional configuration post installation
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/vertical-isp/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/vertical-isp/issues/new?body=module:%20base_phone_rate_import_bandwith%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Open Source Integrators
+
+Contributors
+~~~~~~~~~~~~
+
+* Maxime Chambreuil <mchambreuil@opensourceintegrators.com>
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+.. |maintainer-wolfhall| image:: https://github.com/wolfhall.png?size=40px
+    :target: https://github.com/wolfhall
+    :alt: wolfhall
+.. |maintainer-max3903| image:: https://github.com/max3903.png?size=40px
+    :target: https://github.com/max3903
+    :alt: max3903
+
+Current `maintainers <https://odoo-community.org/page/maintainer-role>`__:
+
+|maintainer-wolfhall| |maintainer-max3903| 
+
+This module is part of the `OCA/vertical-isp <https://github.com/OCA/vertical-isp/tree/12.0/base_phone_rate_import_bandwith>`_ project on GitHub.
+
+You are welcome to contribute. To learn how please visit https://odoo-community.org/page/Contribute.
```

Comment:
It's easier to use the `readme` fragments subdirectory, and put here a README placeholder file.